iroha-cpp/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+set(EXECUTABLE_OUTPUT_PATH ${PROJECT_BINARY_DIR}/bin)
+
+add_subdirectory(services)
+
+add_library(grpcclient
+    GrpcClient.cpp
+    QueryResponseHandler.cpp
+    TransactionResponseHandler.cpp
+    GrpcResponseHandler.cpp
+    GrpcClient.hpp
+    QueryResponseHandler.hpp
+    TransactionResponseHandler.hpp
+    GrpcResponseHandler.hpp
+)
+
+target_link_libraries(grpcclient
+    grpc_channel_factory
+    model_generators
+    command_client
+    query_client
+)
+
+target_include_directories(grpcclient PUBLIC
+    ${PROJECT_SOURCE_DIR}/iroha-cpp
+)
+
+add_executable(iroha-cpp
```

Comment:
Here should be separate binary for each example. Rather name should be different

Reply:
Outdated comment.